Xamarin.Essentials/Sms/Sms.android.cs

```diff
@@ -24,16 +25,15 @@ static Task PlatformComposeAsync(SmsMessage message)
         }
 
         static Intent CreateIntent(SmsMessage message)
-            => CreateIntent(message?.Recipient, message?.Body);
+            => CreateIntent(message?.Recipients, message?.Body);
 
-        static Intent CreateIntent(string recipient, string body = null)
+        static Intent CreateIntent(string[] recipients, string body = null)
```

Comment:
can we change this to params at the end? means changing the body to not be default null. This means you don't need to create the new [] in the IsComposeSupported

Reply:
Done